src/com/zalinius/zje/physics/Vertex.java

```diff
@@ -0,0 +1,49 @@
+package com.zalinius.zje.physics;
+
+import java.awt.geom.Point2D;
+
+public class Vertex implements Physical{
+	private Point position;
+	private Vector velocity;
+	private double mass;
+	
+	public Vertex(double x, double y) {
+		this(new Point(x, y));
+	}
+	public Vertex(Point position) {
+		this(position, 1);
+	}
+	public Vertex(Point position, double mass) {
+		this.position = position;
+		velocity = new Vector();
+		this.mass = mass;		
+	}
+	
+	/**
+	 * Adds an impulse (instantaneous change of momentum) to a vertex
+	 * @param momentum in kg*px/s
```

Comment:
Will pixel momentum cause funkiness with different resolutions and such?

Reply:
Ah, bad comments on my part which remained from Polygon Pal, the physics math is graphics-agnostic